src/lib.rs

```diff
@@ -538,12 +539,26 @@ pub fn filename_to_module(filename: &str) -> StdString {
     name.replace("/", ".")
 }
 
+fn get_env_path() -> Option<String> {
+    match env::var("GLUON_PATH") {
+        Ok(val) => Some(val),
+        Err(_) => None,
+    }
+}
+
 /// Creates a new virtual machine with support for importing other modules and with all primitives
 /// loaded.
 pub fn new_vm() -> RootedThread {
     let vm = RootedThread::new();
-    vm.get_macros().insert(String::from("import"),
-                           ::import::Import::new(::import::DefaultImporter));
+    let gluon_path = get_env_path().unwrap_or(String::from("."));
+    vm.get_macros()
+        .insert_and_get(String::from("import"),
+                        ::import::Import::new(::import::DefaultImporter))
+        .as_ref()
+        .and_then(|import| import.downcast_ref::<::import::Import>())
+        .expect("Import macro")
+        .add_path(gluon_path);
+
```

Comment:
Might also be cleaner to import the types at the top of the function. Generally in Rust the convention is to import types directly, and keep functions qualified by their parent module.